app/helpers/parse/slugify.spec.js

```diff
@@ -0,0 +1,11 @@
+const assert = require('assert')
+
+const slugify = require('./slugify')
+
+describe('slugify', function () {
+    it('should return slug correctly', function () {
+        assert.equal('slug-teste', slugify('SLUG TESTE'))
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/UFABCNextOps/ufabc-next-server/pullRequest?prid=8443142)
```suggestion
    assert.equal('slug-teste', slugify('SLUG TESTE'))
```